transaction/extra/src/unsigned_tx.rs

```diff
@@ -11,7 +10,9 @@ use mc_transaction_core::{
     tx::{Tx, TxPrefix},
     FeeMap,
 };
+use mc_transaction_summary::{TxOutSummaryUnblindingData, TxSummaryUnblindingData};
 use mc_transaction_types::{Amount, BlockVersion, TokenId, TxSummary, UnmaskedAmount};
+
```

Comment:
our style-guide says not to insert blank lines like this, we want cargo fmt to sort them